openqaoa/devices.py

```diff
@@ -172,27 +169,12 @@ def _check_provider_connection(self) -> bool:
         """
 
         try:
-            if IBMQ.active_account() is None:
-                self.provider = IBMQ.enable_account(self.api_token, hub=self.hub,
-                                                    group=self.group,
-                                                    project=self.project)
-            elif IBMQ.active_account()['token'] != self.api_token:
-                IBMQ.disable_account()
-                self.provider = IBMQ.enable_account(self.api_token, hub=self.hub,
-                                                    group=self.group,
-                                                    project=self.project)
-            else:
-                self.provider = IBMQ.get_provider(hub=self.hub, group=self.group,
-                                                  project=self.project)
-
+            self.provider = IBMQ.load_account()
+            if any([self.hub, self.group, self.project]):
+                self.provider = IBMQ.get_provider(hub=self.hub, group=self.group, project=self.project)
             return True
-        
-        except RequestsApiError as e:
-            print('The api key used was invalid: {}'.format(e))
-            return False
-        
         except Exception as e:
-            print('An Exception has occured when trying to connect with the provider: {}'.format(e))
+            print('An Exception has occured when trying to connect with the provider. Please note that you are required to set up your IBMQ account locally first.: {}'.format(e))
```

Comment:
shall we provide a link here to how users can set it up? A link referring to qiskit documentation on how to save token locally?

Reply:
Added in lastest commit